Podfile

```diff
@@ -5,8 +5,8 @@ target 'TrustKeystore' do
 
   pod 'BigInt'
   pod 'CryptoSwift', '~> 0.8.1'
-  pod 'TrezorCrypto', inhibit_warnings: true
-  pod 'TrustCore', inhibit_warnings: true
+  pod 'TrezorCrypto', :inhibit_warnings => true
+  pod 'TrustCore', :inhibit_warnings => true
```

Comment:
Both of these are valid

Reply:
Oh, I didn't know. Either way its probably better to keep consistent with the guide.